### PR TITLE
TN-351 formatting for intervention.py, intervention_strategies.py, lazy.py, locale.py

### PR DESCRIPTION
**Pull request by John0Randolph**

### CODE REVIEW

File: portal/models/intervention_strategies.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def user_registered_with_any_clinics(intervention, user):
                  intervention=intervention.name)
             return True
 
-    return user_registered_with_all_clinics if combinator == 'all' else\
+    return user_registered_with_all_clinics if combinator == 'all' else \
```
Comment (pbugni) [inline portal/models/intervention_strategies.py]: Kill the `\` with parens, please.

File: portal/models/intervention_strategies.py [hunk 2]
```diff
@@ -505,8 +506,8 @@ def completed_card_html(assessment_status):
                 ))
             db.session.commit()
         else:
-            if ui.card_html != card_html or ui.link_label != link_label or\
-               ui.link_url != link_url:
+            if ui.card_html != card_html or ui.link_label != link_label or \
```
Comment (pbugni) [inline portal/models/intervention_strategies.py]: Same request to kill the use of `\`

**Comment by John0Randolph** [comment]
@ivan-c when you say "please don't indent more than one level at once" are you referring to line 688 of `intervention_strategies.py`? If so, pep8 wants the double indent - should I ignore and only do a single indent?

**Comment by ivan-c** [comment]
After merging develop in, I'm not seeing any changes. I think they must've been merged in before; closing

**Pull request closed without merging**